src/log-io.js

```diff
@@ -53,19 +75,34 @@ class LogIO {
     }
   }
 
-  static async fromEntryHash (ipfs, entryHash, id, length = -1, exclude, onProgressCallback) {
+  /**
+     * Create a log from a multihash.
+     * @param {IPFS} ipfs An IPFS instance
+     * @param {string} multihash Multihash (as a Base58 encoded string) to create the Log from
+     * @param {number} [length=-1] How many items to include in the log
+     * @param {Array<Entry>} [exclude] Entries to not fetch (cached)
+     * @param {function(cid, entry, parent, depth)} onProgressCallback
+     * @returns {Promise<Log>}
+     * @deprecated
+     */
+  static async fromMultihash (ipfs, multihash, length = -1, exclude, onProgressCallback) {
+    if (!isDefined(ipfs)) throw LogError.IPFSNotDefinedError()
+    if (!isDefined(multihash)) throw new Error(`Invalid multihash: ${multihash}`)
+
+    return LogIO.fromCID(ipfs, multihash, length, exclude, onProgressCallback)
+  }
+
+  static async fromEntryCid (ipfs, entryCid, length = -1, exclude, onProgressCallback) {
     if (!isDefined(ipfs)) throw LogError.IpfsNotDefinedError()
-    if (!isDefined(entryHash)) throw new Error("'entryHash' must be defined")
+    if (!isDefined(entryCid)) throw new Error("'entryCid' must be defined")
 
-    // Convert input hash(es) to an array
-    const entryHashes = Array.isArray(entryHash) ? entryHash : [entryHash]
```

Comment:
This is the code I think it's bogus. This will cause the EntryIO `cache` object to be  `{ undefined: 'zd..' }`

Reply:
Solved by @shamb0t